tests/unittests/test_fragment_uri.cc

```diff
@@ -1,10 +1,6 @@
-/* -*- tab-width: 4; -*- */
-/* vi: set sw=2 ts=4 expandtab: */
-
-/*
- * Copyright 2010-2020 Mark Callow.
- * SPDX-License-Identifier: Apache-2.0
- */
+// Copyright 2022-2023 The Khronos Group Inc.
```

Comment:
Please keep the emacs and vim modelines. You can change the comment style or even put them at the end of the file if you want.